### PR TITLE
Do not inferParser for ignored files

### PR DESCRIPTION
**Pull request by fisker**
`inferredParser` is useless for ignored files.

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
Strange, those codes has test covered, but coverage says not.

File: src/common/get-file-info.js [hunk 1]
```diff
@@ -67,27 +67,38 @@ function _getFileInfo({
 }) {
   const fileInfo = {
     ignored: ignorer.ignores(filePath),
-    inferredParser: options.inferParser(filePath, plugins) || null,
+    inferredParser: null,
   };
 
-  if (!fileInfo.inferredParser && resolveConfig) {
```
Comment (thorn0) [inline src/common/get-file-info.js:73]: Looks like a bug. `parser` can be overridden by the config.

Comment (fisker) [reply src/common/get-file-info.js:73]: Yes, I noticed this.

Comment (fisker) [reply src/common/get-file-info.js:73]: Added a test, I'll fix it in another PR.

**Pull request merged**